docs/pt-br/actions.md

```diff
@@ -0,0 +1,185 @@
+# Ações
+
+Ações são similares às mutações, sendo que as diferenças são:
+
+- Ao invés de mudar os estados, ações cometem mutações.
```

Comment:
Should we use "cometer" as translation for "commit"? Maybe "confirmar" makes more sense in portuguese.